### PR TITLE
ASC 4.0 Release Notes

### PR DESCRIPTION
**Pull request by hickskenh**
## Purpose of this pull request

This pull request (PR) adds content for the 4.0 release. It also updates the Release page.

## Affected DevDocs pages

-  https://devdocs.magento.com/extensions/amazon-sales/release-notes/
- https://devdocs.magento.com/release/

whatsnew
Available now! Amazon Sales Channel 4.0 introduces a new [Amazon Sales Channel](https://docs.magento.com/m2/ce/user_guide/sales-channels/asc/amazon-sales-channel-home.html) home page, a new [store dashboard](https://docs.magento.com/m2/ce/user_guide/sales-channels/asc/amazon-store-dashboard.html), a streamlined [onboarding process](https://docs.magento.com/m2/ce/user_guide/sales-channels/asc/amazon-onboarding-home.html), and new [default store settings](https://docs.magento.com/m2/ee/user_guide/sales-channels/asc/default-store-settings.html).

### CODE REVIEW

File: src/extensions/amazon-sales/index.md [hunk 1]
```diff
@@ -16,72 +19,51 @@ The Amazon Sales Channel extension installs and adds features to integrate your
...
+1. Export the contents to your Magento root.
 
-   You may need to login with your Magento web account credentials. If you need to create an account, visit [here](https://account.magento.com/customer/account/login) and register.
+1. Follow all instructions for Extension [Installation]({{ site.baseurl }}/extensions/install/).
 
-1. API keys are available through the **API Portal** tab. Copy the key.
+1. Update the services connector entry in your `composer.json` file.
 
-   If you need to create a new key, enter a description like "Amazon Sales Channel" and click **Add**. Copy this new key.
+   -  Add the extension's name and version to your composer.json file:
```
Comment (jeff-matthews) [inline src/extensions/amazon-sales/index.md]:
```suggestion
   -  Add the extension's name and version to your composer.json file.
```

File: src/extensions/amazon-sales/index.md [hunk 2]
```diff
@@ -3,11 +3,14 @@ group: extensions
...
 
+{:.bs-callout .bs-callout-warning}
+Amazon Sales Channel versions 4.0+ are only supported when integrated with Magento 2.3.x.
+
 The Amazon Sales Channel extension installs and adds features to integrate your Magento catalog with Amazon Seller Accounts to sell through the Amazon Marketplace. To review additional information, see the [Amazon Sales Channel Marketplace](http://marketplace.magento.com/magento-module-amazon.html) page.
 
 ## Requirements
 
--  **Magento Instance**: Amazon Sales Channel can be installed on instances with {{site.data.var.ce}} and {{site.data.var.ee}} versions 2.2.4+ and 2.3.X. We do not support the extension on Magento 2.1 or Magento 1.
+-  **Magento Instance**: Amazon Sales Channel can be installed on instances with {{site.data.var.ce}} and {{site.data.var.ee}} versions 2.3.x. We no longer support the extension on Magento 2.1, Magento 2.2, or Magento 1.
```
Comment (jeff-matthews) [inline src/extensions/amazon-sales/index.md]: The release notes indicate support for ece as well. Should that be added here?

File: src/extensions/amazon-sales/release-notes/index.md [hunk 3]
```diff
@@ -25,13 +45,13 @@ Amazon Sales Channel 3.0.1 is compatible with versions 2.2.4+ and 2.3.x of {{sit
...
 
 ### v3.0.0
 
 Amazon Sales Channel 3.0.0 is compatible with versions 2.2.4+ and 2.3.x of {{site.data.var.ce}}, {{site.data.var.ee}}, and {{site.data.var.ece}}.
 
--  {:.new}**Amazon UK Marketplace Now Available**: Users can choose the United Kingdom marketplace when [creating and integrating](https://docs.magento.com/m2/ce/user_guide/sales-channels/amazon/store-integration.html) an Amazon Sales Channel store. This UK upgrade includes additional support for:
+-  {:.new}**Amazon UK Marketplace Now Available**: Users can choose the United Kingdom marketplace when [creating and integrating](https://docs.magento.com/m2/ce/user_guide/sales-channels/asc/store-integration.html) an Amazon Sales Channel store. This UK upgrade includes additional support for:
```
Comment (leslietilling) [inline src/extensions/amazon-sales/release-notes/index.md]: same link issue here

File: src/extensions/amazon-sales/release-notes/index.md [hunk 4]
```diff
@@ -41,7 +61,7 @@ Amazon Sales Channel 3.0.0 is compatible with versions 2.2.4+ and 2.3.x of {{sit
 
 -  {:.fix}**Product Catalog**: <!--CHAN-3687-->Corrected an issue preventing images imported with an Amazon listing from being applied to the corresponding Magento catalog product.
 
--  {:.fix}**Order Creation**: <!--CHAN-3708-->Corrected an issue preventing Magento from creating orders for an Amazon order that does not match with a Magento catalog product. See [Order Settings](https://docs.magento.com/m2/ce/user_guide/sales-channels/amazon/order-settings.html).
+-  {:.fix}**Order Creation**: <!--CHAN-3708-->Corrected an issue preventing Magento from creating orders for an Amazon order that does not match with a Magento catalog product. See [Order Settings](https://docs.magento.com/m2/ce/user_guide/sales-channels/asc/order-settings.html).
```
Comment (leslietilling) [inline src/extensions/amazon-sales/release-notes/index.md]: same link issue here

File: src/extensions/amazon-sales/index.md (hunk 2 above)
Comment (hickskenh) [reply src/extensions/amazon-sales/index.md]: updated to match both places

Comment (jeff-matthews) [reply src/extensions/amazon-sales/index.md]: I don't see any change.

Comment (hickskenh) [reply src/extensions/amazon-sales/index.md]: i haven't pushed the commit yet

File: src/extensions/amazon-sales/index.md [hunk 5]
```diff
@@ -16,72 +19,51 @@ The Amazon Sales Channel extension installs and adds features to integrate your
...
+     ```
 
-   A store configuration page opens to **Stores** > **Configuration** > **Services** > **Magento Services**.
+   -  Enter your [authentication keys]({{ site.baseurl }}/guides/v{{ site.version }}/install-gde/prereq/connect-auth.html). Your public key is your username; your private key is your password.
 
-1. On the Api Keys screen, paste the key you copied for **Production API Key**.
+   -  Wait for Composer to finish updating your project dependencies and make sure there aren't any errors.
 
-1. Click **Save Config**.
+1. After installing, enter an [API Key](https://docs.magento.com/m2/ce/user_guide/sales-channels/asc/amazon-verify-api-key.html) to complete configuration.
```
[the remark this answers is not in the document]
Comment (hickskenh) [reply src/extensions/amazon-sales/index.md]: corrected to ee

File: src/extensions/amazon-sales/index.md [hunk 6]
```diff
@@ -16,72 +19,51 @@ The Amazon Sales Channel extension installs and adds features to integrate your
...
 
-1. On the Admin sidebar, click **Marketing**. Then under **Channels**, click **Amazon**. Magento verifies and validates the entered API key, continuing with onboarding.
+## Add the Amazon Sales Channel API key
 
-See [Onboarding Amazon Sales Channel](https://docs.magento.com/m2/ce/user_guide/sales-channels/amazon/amazon-onboarding-home.html) to continue.
+See [Add or Verify the Amazon API Key](https://docs.magento.com/m2/ee/user_guide/sales-channels/asc/amazon-verify-api-key.html) to continue.
```
[the remark this answers is not in the document]
Comment (hickskenh) [reply src/extensions/amazon-sales/index.md]: corrected to ee

File: src/extensions/amazon-sales/release-notes/index.md [hunk 7]
```diff
@@ -2,12 +2,14 @@
...
+{:.bs-callout .bs-callout-warning}
+Amazon Sales Channel versions 4.0+ are only supported when integrated with Magento 2.3.x.
 
-**Amazon Sales Channel** is now available for version 2.2.x and 2.3.x of {{site.data.var.ce}}, {{site.data.var.ee}}, and {{site.data.var.ece}}. Merchants can use Amazon Sales Channel to integrate with an Amazon Seller account to post and sell products in your Magento catalog on Amazon.
+**Amazon Sales Channel** is available for version 2.3.x of {{site.data.var.ce}}, {{site.data.var.ee}}, and {{site.data.var.ece}}. Merchants can use Amazon Sales Channel to integrate with an Amazon Seller account to list and sell products from your Magento catalog on Amazon.
 
 See the following documentation:
 
--  [Amazon Sales Channel](https://docs.magento.com/m2/ce/user_guide/sales-channels/amazon/amazon-sales-channel.html) for merchant information and instructions
+-  [Amazon Sales Channel](https://docs.magento.com/m2/ce/user_guide/sales-channels/asc/amazon-sales-channel.html) for merchant information and instructions
```
[the remark this answers is not in the document]
Comment (hickskenh) [reply src/extensions/amazon-sales/release-notes/index.md]: corrected to ee

File: src/extensions/amazon-sales/release-notes/index.md (hunk 3 above)
Comment (hickskenh) [reply src/extensions/amazon-sales/release-notes/index.md]: corrected to ee

File: src/extensions/amazon-sales/release-notes/index.md (hunk 4 above)
Comment (hickskenh) [reply src/extensions/amazon-sales/release-notes/index.md]: corrected to ee

File: src/extensions/amazon-sales/index.md (hunk 5 above)
[the remark this answers is not in the document]
Comment (leslietilling) [reply src/extensions/amazon-sales/index.md]: still says `ce`... hmmmm

[the remark this answers is not in the document]
Comment (hickskenh) [reply src/extensions/amazon-sales/index.md]: it still says ce in this view.  if you click "edit file", the ee shows in the source.

**Comment by hickskenh** [comment]
running tests

**Pull request merged**